qCC/ccGraphicalSegmentationTool.cpp

```diff
@@ -49,1600 +50,1616 @@
 #include <QMessageBox>
 #include <QPushButton>
 #include <QInputDialog>
+#include <QSettings>
 
 //System
 #include <assert.h>
 
-ccGraphicalSegmentationTool::ccGraphicalSegmentationTool(QWidget* parent)
-	: ccOverlayDialog(parent)
-	, Ui::GraphicalSegmentationDlg()
-	, m_somethingHasChanged(false)
-	, m_state(0)
-	, m_segmentationPoly(nullptr)
-	, m_polyVertices(nullptr)
-	, m_rectangularSelection(false)
-	, m_deleteHiddenParts(false)
+ccGraphicalSegmentationTool::ccGraphicalSegmentationTool(QWidget *parent)
```

Comment:
nope 😈 (we keep the 'start' next to the type in this project)

And it seems you have changed the alignment of the initializers (we use real tabs)